ci/prow/check-commits-count

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+number_of_commits=$(git rev-list --count HEAD ^main)
+if [[ ${number_of_commits} != 1 ]]; then
```

Comment:
Did you test it? Why don't we check the last 2 commits like we do u/s?
https://github.com/kubernetes-sigs/kernel-module-management/pull/69/files

Reply:
You're right. Modified.